avm/ptn/azd/monitoring/main.bicep

```
@@ -0,0 +1,100 @@
+metadata name = 'Azure Monitoring'
```

I'd suggest adding `Azd` prefix to the title. This would need a quick readme regeneration
```suggestion
metadata name = 'Azd Azure Monitoring'
```

---

@eriqua  I have updated by your comments.